src/api/model.js

```diff
@@ -1,18 +1,30 @@
+import fs from 'fs';
 import express from 'express';
 import { addAction } from '../lib/database';
 import ModelStore from '../store/model-store';
 import ItemStore from '../store/item-store';
 
 let app = express();
 
+function getModelPhoto(address) {
+    let bitmap = fs.readFileSync(ModelStore.getPhotoPath(address));
+    return Buffer.from(bitmap).toString('base64');
+}
+
 app.get('/', (req, res) => {
     let model = ModelStore.getModelByAddress(req.query.address);
+    model.photo = getModelPhoto(model.address);
     res.successJson(model);
 });
 
 app.get('/all', (req, res) => {
+    let models = ModelStore.getModels();
+
+    models.forEach(model => {
+        model.photo = getModelPhoto(model.address);
```

Comment:
Aren't we storing images as files in order to avoid having them all in memory at once? This is loading every single model photo into memory at once.